contracts/ExtendedFinance.sol

```diff
@@ -35,4 +37,28 @@ contract ExtendedFinance is BaseFinance {
 
     }
 
+    /**
+    @notice Withdraws user's rewards
+    @dev
+        Function to transfer a certain user's rewards to his address 
+    @param _user Address who's rewards will be withdrawn
+    @param _value Value of the withdraws which will be transfered to the user 
+    */
+    function withdrawRewards(address _user, uint256 _value) public onlyOwnerOrAllowed {
+        require(rewardedBalance[_user] >= _value);
+
+        appc.transfer(_user, _value);
```

Comment:
This can be prone to the known [_Reentrancy_ problem](https://consensys.github.io/smart-contract-best-practices/known_attacks/). Shouldn't we update the rewardedBalance before transferring the APPC?
```suggestion
        rewardedBalance[_user] -= _value;
        appc.transfer(_user, _value);
```

Reply:
Fixed